lib/todo/menu_options_formatter.ex

```diff
@@ -1,18 +1,33 @@
 defmodule ToDo.MenuOptionsFormatter do
   def menu_options_for_display do
-    Enum.map_join(menu_options, "\n", &("#{&1}"))
+    menu_options_content
+    |> prepare_menu_options
+    |> format_for_display
     |> append_newline
   end
 
-  defp menu_options do
-      [
-      "Commands Cheatsheet:",
-      "[aA] Add New To-Do Items",
-      "[bB] List All Items",
-      "[cC<item_number>] Edit Item",
-      "[dD<item_number>] Delete Item",
-      "[eE] Exit"
-      ]
+  def menu_options_content do
+    [ %{downcase_command: "a", upcase_command: "A", task_number: false, description: "Add New To-Do Tasks"},
+      %{downcase_command: "b", upcase_command: "B", task_number: false, description: "List All Tasks"},
+      %{downcase_command: "c", upcase_command: "C", task_number: true, description: "Edit Task"},
+      %{downcase_command: "d", upcase_command: "D", task_number: true, description: "Delete Task"},
+      %{downcase_command: "e", upcase_command: "E", task_number: false, description: "Exit"}
+    ]
+  end
+
+  def prepare_menu_options(options) do
+    ["Commands Cheatsheet:"] ++
+    Enum.map(options, fn(option) ->
+      case option do
+        %{downcase_command: lc, upcase_command: uc, task_number: false, description: d} -> "[#{lc}#{uc}] #{d}"
```

Comment:
With maps you can pattern match on only the things you need, so in this case `task_number` can be omitted.